wiki/Tournaments/OWC/2013/de.md

```diff
@@ -409,7 +409,7 @@ Das Turnier kam zu einem Ende mit folgenden Platzierungen:
 1. Das osu! World Cup ist ein länderbezogenes 4 gegen 4 Team Turnier.
 2. Die Beatmaps werden vom Map Wähler für jede Runde im Vorhinein am Sonntag angekündigt, bevor die eigentliche Partie stattfindet. Nur diese werden dann während den eigentlichen Partien zur Verfügung stehen.
    - Eine Beatmap wird als Tiebreaker-Map mit angegeben. Diese Beatmap wird nur beim größtmöglichem Punktegleichstand gespielt.
-   - Es wird auch einen [Hidden](/wiki/Game_modifier/Hidden), [HardRock](/wiki/Game_modifier/Hard_Rock), [DoubleTime](/wiki/Game_modifier/Double_Time) und FreeMod Stappel geben.
+   - Es wird auch einen [Hidden](/wiki/Game_modifier/Hidden), [HardRock](/wiki/Game_modifier/Hard_Rock), [DoubleTime](/wiki/Game_modifier/Double_Time) und FreeMod Sapel geben.
 3. Die Zeiten der Partien werden vom Turnier Management festgelegt (siehe unten).
 4. Wenn kein Schiedrichter oder Personal vorhanden sein sollte, wird die Partie verschoben.
 5. Die Ergebnisse von wiederbelebten Spielern wird zum Teamergebnis mit angerechnet.
```

Comment:
```suggestion
5. Die Ergebnisse von wiederbelebten Spielern werden zum Teamergebnis mit angerechnet.
```

---

There are a few things further below, which I can't edit myself unfortunately. These are as follows:

- 420: Turnier-Management
- 423: Du anstatt Sie
- 424: Not sure if `nichtigen` is a word
- 425: fehlendes `angesehen`; maybe something like `Unterschiedverhältnis` for difference ratio?
- 426: Turnier-Management; There is also a missing sentence here: `Die Entscheidung eines Schiedrichters kann vom Turnier-Management überschrieben werden.`

Please recheck this section yourself, just to make sure no sentences are left out (as the above one).

Reply:
I changed the things you pointed out here. I will read through this section once more tomorrow to hopefully make sure there are no further issues. :sweat: